lib/org/chromium/webidl/PipelineMessage.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
I'm not entirely sure what you mean by "identify" the messages. We should probably be using metadata in our `Runnable` implementations that specify input and output types (and possibly add add instanceOf() checks on inputs and outputs).

Do you mean how will we support tracing messages from one stage of the pipeline to the next? There are a few ways of doing it. Abstractly, each output should contain just enough "extra" information to uniquely identify the input that it came from (in addition to the output data itself, including identifying information). Perhaps we should discuss concrete strategies offline.